components/api/src/main/java/com/hotels/styx/api/UrlQuery.java

```diff
@@ -102,10 +101,13 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return toStringHelper(this)
-                .add("parameters", parameters)
-                .add("encodedQuery", encodedQuery)
-                .toString();
+        StringBuilder sb = new StringBuilder(64);
```

Comment:
Purly subjective but you could chain the invocations together in a fluent style. IMO would look nicer.

Reply:
You are right, and much as I'd love to say "but there are so many to change" I think that's actually an argument for getting it done. So I'll get it done.

Reply:
Hi guys. I believe there are many classes which do not frankly deserve a `toString` method. I would only convert for true data classes.